packages/syft/tests/syft/core/tensor/autodp/single_entity_phi_test.py

```diff
@@ -1181,6 +1181,68 @@ def test_clip() -> None:
     assert (clipped_tensor3 >= rand1).all()
 
 
+# tensor filled with zeros
+zeros_tensor = SEPT(
```

Comment:
Could we replace these new global variables and implement them as pytest fixtures instead? (I think we discussed this during the meeting today)
You could merge from the `dev_fixed_adp_tests` branch and make use of some of the ones already implemented :)

Reply:
+1